src/test/java/org/junit/runners/CustomBlockJUnit4ClassRunnerTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2002-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
JUnit isn't licensed under the Apache License. Please remove this entire comment block